metatron-app/metatron-discovery
# Fixed error in dark theme

### Description

1. When the theme is dark, the legend text is not visible in the chart editing screen.
2. The paging text of the chart legend is invisible when dark themes.
3. I can't see the Edit Description button for Wrangled Dataset when it is dark.
4. When the theme is dark, the text is not visible when editing the title.

**Related Issue** : 


### How Has This Been Tested?

1. Check that the text is displayed correctly when the legend is displayed on the chart editing screen.
2. Verify that the paging text of the chart legend is displayed.
3. Under Preferences, verify that the Modify Description button for the Wrangled Dataset is displayed.
4. Make sure the text is visible when editing the title / description.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
